homeassistant/components/axis/binary_sensor.py

```diff
@@ -35,23 +35,29 @@ def __init__(self, event, device):
         """Initialize the Axis binary sensor."""
         self.event = event
         self.device = device
-        self.delay = device.config_entry.options[CONF_TRIGGER_TIME]
         self.remove_timer = None
+        self.unsub_dispatcher = None
```

Comment:
We're not unloading the config entry yet, right? So this isn't used yet.

Reply:
Getting there! Its been a part of the previous two PRs

Reply:
Ok. Just checking so I'm not missing anything. I wanted to check that we handle a single listener and a list of listeners correctly.

Reply:
Impressive detail to catch 👌